db/src/main/kotlin/edu/artic/db/models/ArticTour.kt

```diff
@@ -131,6 +132,29 @@ data class ArticTour(
         return this.title
     }
 
+    /**
+     * Returns the translations including english language
+     */
+    val translationWithEnglish: List<Translation>
```

Comment:
Please rename to `allTranslations` to match the naming used by `ArticAudioFile`.

Reply:
Refactored.